modules/mapbox/src/mapbox-overlay.ts

```diff
@@ -71,10 +77,15 @@ export default class MapboxOverlay implements IControl {
     });
     this._container = container;
 
-    this._deck = new Deck({
-      ...this._props,
-      parent: container,
-      viewState: getViewState(map)
+    this._deck = getDeckInstance({
```

Comment:
These props also need to set in the overlaid use case in the event `layers: []`

Reply:
Overlaid mode should not call `getDeckInstance`. There is no context sharing and deck canvas is not dependent on Mapbox behaviors.